app/api/government_service_data_api/calls_received_metric.rb

```diff
@@ -47,6 +47,26 @@ def not_provided?
     ].all? { |item| item == NOT_PROVIDED }
   end
 
+  def unspecified
+    return NOT_APPLICABLE if not_applicable? || not_provided?
+
+    value = @total - [@get_information,
```

Comment:
In the case where we have a total, but none of the breakdowns are supplied, what should we do? If I've understood this code correctly, it'll have the entire total be "unspecified". Is that the behaviour we want?

The other way of handling it I can think of is to return `NOT_PROVIDED` so it reads as "has total, no breakdown provided".

Reply:
I think it might be more logical to read it as "Total was 100, all of it unspecified".  If we start accepting that 'unspecified' can be 'not_provided' the logic starts getting a bit more complicated.

Reply:
So in the view we'd get the total as the headline figure, the breakdown table with everything marked as "Not provided" and then the total figure repeated in "Unspecified"?

I'm not really sure what the best approach is here, but as long as we've considered it, I'm OK with any solution.

Reply:
We can test it, see what people think when they see it.

Reply:
👍